packages/blade/src/components/Box/styledProps/getStyledProps.ts

```diff
@@ -31,7 +31,8 @@ const removeUndefinedStyledProps = (obj: StyledPropsInputType): StyledPropsInput
   const onlyDefinedStyledProps: StyledPropsBlade = {};
   for (const key in obj) {
     if (obj[key as keyof StyledPropsBlade] !== undefined) {
-      // @ts-expect-error: complex type error
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
```

Comment:
why did we replaced tsexpecterror with tsignore tho?

Reply:
Better thicc than skinny

Reply:
because ts-expect-error goes unused in react native typecheck but used in web typecheck. So on react-native, it throws unused error on cli.

<img width="349" alt="image" src="https://user-images.githubusercontent.com/30949385/231704851-756f9aed-9912-4edb-b8d6-fc41ff61ac00.png">